lib/vanagon/project/dsl.rb

```diff
@@ -268,6 +268,12 @@ def retry_count(retry_count)
         @project.retry_count = retry_count
         environment 'VANAGON_PROJECT_RETRY_COUNT', @project.retry_count
       end
+
+      def package_override(var)
+        platform = @project.platform
+        fail "I don't know how to set package overrides for #{platform.name}, teach me?" unless platform.is_rpm? || platform.is_deb?
```

Comment:
This is good enough for a rev. 1 but I'd lose the `unless` conditional here and move `@project.package_overrides << var` directly into any platforms that support it ASAP. Or do you think we're better served just doing it now?